modules/tinymce/src/core/test/ts/browser/dom/SelectionTest.ts

```diff
@@ -1195,6 +1195,15 @@ describe('browser.tinymce.core.dom.SelectionTest', () => {
     LegacyUnit.equal(curRng.endContainer.nodeName, 'P');
     LegacyUnit.equal(curRng.endOffset, 0);
   });
+
+  it('TINY-9001: Expanding a word expands the selection', () => {
```

Comment:
Maybe also stick in a test where we actively pass in the config object with word. Since when we implement that in the future mistakes could be made. That also goes for the RangeUtils function add a explicit test for that.